src/itype.h

```diff
@@ -255,6 +255,20 @@ struct islot_book {
     recipe_list_t recipes;
 };
 
+struct islot_mod {
+    /** If non-empty restrict mod to items with those base (before modifiers) ammo types */
+    std::set<ammotype> acceptable_ammo;
+
+    /** If set modifies parent ammo to this type */
+    ammotype ammo_modifier = NULL_ID;
```

Comment:
Shit, I merged it because it didn't crash on my build, but `NULL_ID` in unspecified context (possibly static) is still unsafe.